.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "dev/ca8210"]
 	path = dev/ca8210
 	url = https://github.com/Cascoda/ca8210-contiki.git
+[submodule "apps/letmecreateiot"]
```

Comment:
As it is specific to Mikro-e platform, it can be downloaded at platform/mikro-e/apps/letmecreateiot.